internal/database/migration/cliutil/drift_schema.go

```diff
@@ -0,0 +1,58 @@
+package cliutil
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"regexp"
+	"strings"
+
+	descriptions "github.com/sourcegraph/sourcegraph/internal/database/migration/schemas"
+	"github.com/sourcegraph/sourcegraph/lib/errors"
+)
+
+type ExpectedSchemaFactory func(repoName, version string) (descriptions.SchemaDescription, bool, error)
+
+// GCSExpectedSchemaFactory reads schema definitions from a GCS bucket that contains schema definitions for
```

Comment:
```suggestion
// GCSExpectedSchemaFactory reads schema definitions from a public GCS bucket that contains schema definitions for
```

This must be a public bucket right?